java/src/jmri/jmrit/display/layoutEditor/MultiSensorIconFrame.java

```diff
@@ -164,11 +176,11 @@ public String toString() {
         Entry(JPanel self, JmriJFrame frame, String name) {
             this.self = self;
             this.setLayout(new FlowLayout());
-            this.add(new JLabel("Sensor:"));
+            this.add(new JLabel(Bundle.getMessage("BeanNameSensor") + ":"));
```

Comment:
You're totally right, in principle (see [French.About.com](http://french.about.com/library/writing/bl-punctuation.htm))
But that implies we either need a ``LabelFor_AlreadyTranslatedAbove = AlreadyTranslatedAbove  :`` variant for lots of things or we might use just one ``MakeLabel = {0} : `` key in NamedBeansBundle and call it like ``Bundle.getMessage("MakeLabel", "AlreadyTranslatedAbove")``. I would go for method 2, taking the load off translators. Would it make sense to real programmers?